content/docs/getting-started/adding-a-catalog-item/aws-s3/index.md

```diff
@@ -0,0 +1,158 @@
+---
```

Comment:
We should have a bit of text above the code snippet to explain what it is (curl, yaml etc)